l10n_br_account_payment_order/wizards/account_move_line_change.py

```diff
@@ -30,6 +30,7 @@ def default_get(self, fields_list):
     # Muitas opções são permitidas, verificar manual do respectivo CNAB usado.
     change_type = fields.Selection(
         selection=[
+            ("change_discount_date", "Desconto"),
```

Comment:
```suggestion
            ("change_discount_date", "Data de Desconto"),
```
Aqui poderia ser assim